climpred_skeleton/core.py

```diff
@@ -0,0 +1,25 @@
+import xarray as xr
+
+
+class Verification:
+    """Absolute base class. Begins the pipeline of verifying a forecast.
+
+    Holds a prediction that is being verified against observations. Stores
+    helpful base attributes (`nmember`, etc.) and runs checks on input
+    data.
+    """
+
+    def __init__(self, prediction: xr.Dataset, observation: xr.Dataset) -> None:
+        # Check that inputs are xarray objects, convert to dataset.
+        # Convert to `cftime`.
+        self._prediction = prediction
+        self._observation = observation
+        self._nmember = self._prediction['member'].size
```

Comment:
What if there's no explicitly named member dimension?

Reply:
I was going to say that's a required dimension but I guess it's not if they feed in the ensemble mean. We could wrap the member related stuff in a try block or just do `if 'member' in self._prediction.dims`